### ISSUE TITLE
Not able to flash boot image.

### ISSUE THREAD

**Issue by hammeredhank** [badabing2005/PixelFlasher#120]
I'm unable to flash image with Magisk patch created by PixelFlasher.  After device is put into fastboot PixelFlasher is waiting and then times out. 

Timeout: [60] Fastboot could not detect device: 38120DLJH00251 in bootloader mode 

2023-11-20 15:56:03 ERROR: Encountered an error while rebooting to bootloader
Aborting ...

2023-11-20 15:56:03 ERROR: Encountered an error while flashing
Traceback (most recent call last):
  File "Main.py", line 3415, in _on_flash
AttributeError: 'NoneType' object has no attribute 'get_device_state'

2023-11-20 15:56:48 Creating support.zip file ...
Saved support file to: /root/shiba/support_2023-11-20_15-56-44.zip
[support_2023-11-20_15-56-44.zip](https://github.com/badabing2005/PixelFlasher/files/13420421/support_2023-11-20_15-56-44.zip)

**Comment by badabing2005** [comment]
If PixelFlasher is not detecting your device while in bootloader mode, then please make sure you download the latest USB drivers. (Link is available in the Help menu)
While the device is in bootloader mode, `fastboot devices` should list your device, if it doesn't, there's not much PF can do about it.
Typically this is due to drivers.

**Comment by hammeredhank** [comment]
Yes. I opened a shell and executed fastboot devices and it returned the same id as PixelFlasher shows.
Any other ideas?

Thanks

**Comment by badabing2005** [comment]
Enable debug logging, and we should see additional details
![image](https://github.com/badabing2005/PixelFlasher/assets/22603312/f6bb8092-3cda-47c1-b7d8-f018a180c236)

**Comment by fsteccanella** [comment]
I have the same issue, but to me the device appears as "Misc Data"

**Comment by badabing2005** [comment]
The device is not supposed to appear as "Misc Data"

**Comment by fsteccanella** [comment]
I don't want to "steal" the topic to @hammeredhank so please confirm that you are experiencing the same otherwise I'll open a new issue :)

Btw for the time being:
Os: Arch Linux
Device: Pixel 7
USB Drivers: NA
Platform tools: 34.0.4

Thanks!

**Comment by badabing2005** [comment]
> I don't want to "steal" the topic to @hammeredhank so please confirm that you are experiencing the same otherwise I'll open a new issue :)
> 
> Btw for the time being: Os: Arch Linux Device: Pixel 7 USB Drivers: NA Platform tools: 34.0.4
> 
> Thanks!

Please switch to Platform tool 34.0.5, 34.0.4 is known to have issues.

**Comment by hammeredhank** [comment]
Hi.  Thanks all I solved my problem.  I don't know if it will help you.  The release version was very specific about Ubuntu 20 and 22.  So, I am running Debian 10.  I created a Ubuntu 22 Desktop in virtualbox, and I guess that it was not able to reconnect in fastboot mode when the Pixel rebooted.  Probably a virtualbox usb serial problem.  I changed disks in my NUC and installed Ubuntu and everything worked fine.  

Thanks for your help and developing this program.  I hope @fsteccanella is able to solve his problem.

Thanks.
